### DISCUSSION TITLE
Pushing continuously to Listed

### DISCUSSION THREAD

**Discussion by RemiDesgrange** [standardnotes/forum#2545] [Suggestions]
Sometimes I take notes live with standard notes (meeting, conference), and I would like it to be continuously pushed to listed, so coworker or other see "almost" live.

**Comment by atmoio** [comment]
Would be cool indeed..

**Comment by RemiDesgrange** [comment]
I don't know how you could do this. I mean technically without too much effort.

Maybe at first we could have a shortcut, and say "everytime I press it, it publish" it could be the first step ?!

:arrow_up: you notice that I mention shortcut again :yum:
